### ISSUE TITLE
Develop Branch: Fatal Error

### ISSUE THREAD

**Issue by designermonkey** [slimphp/Slim#789]
> Fatal error: Cannot use object of type Closure as array in /Users/designermonkey/Projects/Manneken/build/vendor/slim/slim/Slim/App.php on line 1186

I'm getting this error since the move to Pimple 2, as Pimple is always returning a Closure instead of a Service. I don't now if this is a Pimple error or something we're not doing right with it now it has changed.

**Comment by codeguy** [comment]
Have you pulled all of the latest commits? Unit tests are all green here. Are you adding custom closures to the Slim container for your app?

**Comment by designermonkey** [comment]
I just pulled all the latest commits, and yes, unit tests are all green here too, which is why I logged it.

With regard to the closures front, yes. ;)

I've broken out the default constructor to build each part of the app slightly differently as I needed to call the configuration from a file based system instead of arrays.The closures are all written in the same format that Pimple states, so I really don't know what is going on...

I've pushed the app to https://github.com/manneken-digital/beagle if you want to take a look, as I'm completely stumped on this one.
